modules/img2img.py

```diff
@@ -115,7 +115,7 @@ def process_batch(p, input_dir, output_dir, inpaint_mask_dir, args, to_scale=Fal
 
         for n, processed_image in enumerate(proc.images):
             filename = image_path.stem
-            infotext = proc.infotext(p, n)
+            infotext = proc.infotext(p, 0)

```

Comment:
your implementation will cause wrong metadata to be applyed to the image if batch > 1
I made an alternative implementation https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/12164

Reply:
Thank you very much!